custom.ts

```diff
@@ -733,34 +741,34 @@ namespace gigglebot {
      */
     //% blockId="gigglebot_show_voltage" block="show battery voltage (mv)"
     //% advanced=true
-    export function show_voltage() {
-        let voltage = get_voltage()
+    export function voltageShow() {
+        let voltage = voltageRead()
         basic.showNumber(voltage)
     }
 
     //% blockId="gigglebot_get_firmware" block="firmware version number"
     //% advanced=true
-    export function get_firmware(): number {
+    export function firmwareRead(): number {
```

Comment:
Maybe name this "firmwareVersion" instead of "firmwareRead". It reads the firmware version. It does not do a firmware dump.

Reply:
fixed